sig-node/README.md

```diff
@@ -1,3 +1,49 @@
-[Meeting Notes](https://docs.google.com/document/d/1Ne57gvidMEWXR70OxxnRkYquAoMpt56o75oZtg-OeBg/edit?usp=sharing)
+# SIG Node
 
-[Hangout](https://plus.google.com/hangouts/_/google.com/sig-node-meetup?authuser=0)
\ No newline at end of file
+A Special Interest Group focused on topics related to the Kubernetes node, such as the Kubelet and
+container runtime. It is an open and active community, and we always welcome new additions!
+
+## Goals:
+
+Topics include, but are not limited to:
+
+* Kubelet related features (e.g. Pod lifecycle)
+* Node level performance and scalability (with [sig-scalability](../sig-scalability))
+* Node reliability
+* Node lifecycle management (with [sig-cluster-lifecycle](../sig-cluster-lifecycle))
+* Container runtimes: docker, [rkt](../sig-rktnetes), etc.
+* Images, package management
+* Resource isolation (with [sig-scheduling](../sig-scheduling))
+* Issues related to monitoring (with [sig-instrumentation](../sig-instrumentation))
+* Node level security and Pod isolation (with [sig-auth](../sig-auth))
+* Kernel interactions (to a limited extent)
+* ...
+
+We also work closely with [sig-storage](../sig-storage) and [sig-networking](../sig-networking). As you can see, this is a very cross-functional team!
```

Comment:
changed "resource isolation" to "resource management", and added a section below.